docs/topics/resources/spring-boot/spring-boot-openshift-pom-config-example.xml

```diff
@@ -0,0 +1,38 @@
+<profiles>
+    <profile>
+      <id>licenses</id>
```

Comment:
@metacosm is the `licenses` profile required to be included whenever using RHOAR Spring Boot, or is it only required for boosters? I left the profile in, just in case it tis necessary for legal reasons, but I am OK with removing it from this example if it is not necessary.

Reply:
Boosters need it for generating the license files, which are mandatory. That's all. I think having it in the docs is nothing but confusing.